### DISCUSSION TITLE
Codec Training Code

### DISCUSSION THREAD

**Discussion by yukiarimo** [QwenLM/Qwen3-TTS#18] [General]
### Description

Hello!

Please release fine-tuning and training from scratch your Qwen Codec (audio tokenizer). No WavLM or anything, pure PyTorch audio-in audio-out loss. I want to train from scratch on my 48 kHz LJSpeech-sized dataset!

Thanks!

### Reproduction

Local

### Logs

```shell

```

### Environment Information

N/A

### Known Issue

- [x] The issue hasn't been already addressed in Documentation, Issues, and Discussions.
